enterprise_data/clients.py

```diff
@@ -44,32 +36,34 @@ def get_enterprise_learner(self, user):
         """
         try:
             querystring = {'username': user.username}
-            endpoint = getattr(self, 'enterprise-learner')
-            response = endpoint.get(**querystring)
-        except (HttpClientError, HttpServerError) as exc:
+            url = urljoin(self.API_BASE_URL, 'enterprise-learner')
+            response = self.get(url, params=querystring)
+            response.raise_for_status()
+            data = response.json()
+        except (HTTPError, RequestException) as exc:
```

Comment:
The current handler looks a little bit odd since HTTPError inherits RequestException it could be omitted if we do not need to process both errors differently. 

Did I miss something?

Reply:
updated